service/middleware.go

```diff
@@ -83,3 +97,24 @@ var MiddlewareWithCSRF = func(inner http.Handler) http.Handler {
 
 	return CSRF(Middleware(inner))
 }
+
+func recoverWithSentry(hub *sentry.Hub, w http.ResponseWriter, r *http.Request) {
+	if err := recover(); err != nil {
+		log.Errorf("recover from error: %v", err)
+		debug.PrintStack()
+
+		w.Header().Set("Content-Type", "application/json; charset=UTF-8")
+		e := response.ErrorInternal
+		w.WriteHeader(e.StatusCode)
+
+		// Encode the response as JSON
+		if err := json.NewEncoder(w).Encode(e); err != nil {
+			log.Printf("Error forming the signing response: %v\n", err)
```

Comment:
This log message seems misleading. Maybe it should indicate that we're in the panic recovery middleware?

Reply:
fixed